doc/rst/language/spec/tuples.rst

```diff
@@ -1213,14 +1213,4 @@ Predefined Functions and Methods on Tuples
 
 
 
-.. function:: proc max(type t) where isTupleType(t)
-
-   Returns a tuple of type ``t`` with each component set to the maximum
-   value that can be stored in its position.
-
-
-
-.. function:: proc min(type t) where isTupleType(t)
-
-   Returns a tuple of type ``t`` with each component set to the minimum
-   value that can be stored in its position.
+.. include:: /builtins/ChapelTuple.rst
```

Comment:
It's a shame that doing this drops the where clause output (because of #5950) but I think it's better to include the code directly than hope that this stays up to date with it (though it hopefully won't change after this point).  Still, I think it'd be worth pointing out on the chpldoc issue that this is impacted